### DISCUSSION TITLE
Print Styles

### DISCUSSION THREAD

**Discussion by themire** [ElMassimo/vite_ruby#363] [General]
I am using the "Importing styles from JS" approach which outputs CSS in the following format

`<link rel="stylesheet" href="/vite/assets/filename.css" media="screen" />` 

I would like my styles to be available in print mode too though. What's the best approach? Use `<%= vite_stylesheet_tag 'admin-js-entrypoint', media: 'print' %>` ? 

Thanks!

**Comment by ElMassimo** [comment]
Hi Julian!

If you want styles to be available both for `print` and `screen` you can use `media: "all"`:

```erb
<%= vite_stylesheet_tag 'filename', media: 'all' %>
```

If you are importing styles from JS, then pass `media` to the `vite_javascript_tag` helper instead:

```erb
<%= vite_javascript_tag 'admin-js-entrypoint', media: 'all' %>
```

**Reply by themire** [reply]
Thanks Massimo!
